reference/forms/types/collection.rst

```diff
@@ -229,6 +229,23 @@ you'd use the :doc:`EmailType </reference/forms/types/email>`. If you want
 to embed a collection of some other form, pass the form type class as this
 option (e.g. ``MyFormType::class``).
 
+keep_as_list
+~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+If set to ``true``, the entries will be reindexed so their indexes follow each
```

Comment:
IMO this reads as if the indexes of the collection in the model data will be reindexed. We should somehow make it more clear that this option influences the reindexing of the nested forms names.

Reply:
I tried a rewording. What do you think of this new version?